components/StyledInputTags.js

```diff
@@ -236,6 +247,12 @@ const StyledInputTags = ({ suggestedTags, value, onChange, renderUpdatedTags, de
                               onClick={() => {
                                 addTag(st);
                               }}
+                              onBlur={() => {
+                                if (st === suggestedTags[suggestedTags?.length - 1]) {
```

Comment:
`suggestedTags` is necessarily defined here so we don't need the question mark. 

```suggestion
                                if (st === suggestedTags[suggestedTags.length - 1]) {
```